src/APlayer.js

```diff
@@ -520,7 +525,7 @@ class APlayer {
                 }
             });
 
-            this.audio.addEventListener('pause', () => {
+            let pauseHandler = () => {
```

Comment:
Seems that `pauseHandler` is not re-assigned below, please use `const` instead.

Reply:
Handled with commit 1a4ea7a4ed4cacfc9e6686495e1e5b72a8e0e1cd.